tests/testData/lsst_wfs_error_test_ofc.z4c

```diff
@@ -0,0 +1,9 @@
+# Wavefront error in the basis of Zernike polynomials (Z4-Z22) in um
+# Row 0: R00_SW0
+# Row 1: R04_SW0
+# Row 2: R40_SW0
+# Row 3: R44_SW0
+0.01825211 -0.09129873  0.00627351 -0.01743816 -0.02255052 -0.09191237  0.08130639 -0.03231235  0.00626321  0.00780239 0.00660058 -0.01218201  0.00816416  0.01221569 -0.0280138  0.01196163 -0.06703311  0.06399901 -0.00893908
+0.01824885  0.0913146   0.00626768  0.01743294 -0.02254619 0.09189719  0.08129811 -0.03231231  0.0062603  -0.00779209 0.00658502  0.01218581  0.00816621 -0.01221705 -0.02802573 -0.0119817  -0.06693048 -0.06409968 -0.00893583
+0.01825013  0.09130833  0.00627337 -0.01743358  0.0225466 -0.09190608 -0.08129838 -0.03231162  0.00626275 -0.00779505 0.00658092  0.01218198 -0.00816474  0.01221652  0.02802544 0.01197182  0.06693082  0.06409961 -0.00893513
+0.01825142 -0.09130473  0.00626763  0.01743788  0.02255021 0.0919034  -0.08130649 -0.03231251  0.00626075  0.00779991 0.00660488 -0.0121865  -0.00816518 -0.01221603  0.02801362 -0.01197158  0.06703344 -0.06399904 -0.00893975
```

Comment:
Please add new line at the end of file. This applies to all the `.z4c` and other files.